app/views/late_reservation_mailer/late_reservation_email.html.erb

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta content='text/html; charset=UTF-8' http-equiv='Content-Type' />
+  </head>
+  <body>
+    <h1>Your reservation has not been ordered yet!</h1>
+    <p>
+      We noticed that you had place a reservation to buy ITEM
```

Comment:
make sure to replace these with the appropriate instance variable values, e.g. `...to buy <%= reservation.link %>...`

check out slim's documentation for the equivalents in slim